apex/apex.g4

```diff
@@ -518,7 +518,7 @@ expression
     :   primary
     |   expression '.' GET '(' expressionList? ')'
     |   expression '.' SET '(' expressionList? ')'
-    |   expression '.' Identifier
+    |   expression ('.' | '?.') Identifier
```

Comment:
I afraid it doesn't work since `?.` token is absent in the APEX grammar. You should introduce either the `?.` token:

```g4
QUESTION_DOT: '?.';
```

or use two tokens instead: `'?' '.'`:

```suggestion
    |   expression '?'? '.' Identifier
```


I don't know what option is more correct in terms of APEX specification.